playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -31,18 +31,50 @@ Cloud Build triggers created by terraform scripts from this directory automate s
     - Cloud Build Editor
     - Security Admin
     - Service Account User
+    - Secret Manager Admin
 - [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
-- An existing GCP Bucket to save Terraform state - `state-bucket`
+- An existing GCS Bucket to save Terraform state - `state-bucket`
+- An existing GCS Bucket to store CI privae logs - `private-logs-bucket`
 - DNS name for your Playground deployment instance
 - [Terraform](https://www.terraform.io/)
+- [GitHub Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
 - [Apache Beam GitHub](https://github.com/apache/beam) repository cloned locally
 
 ## 1. Set up the Google Cloud Build for your GCP project
 
-The `playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup` provisions dependencies required to set up Cloud Build for Playground:
+First provide the variables by creating a `common.tfvars` file in the environment folder 
+```
+$BEAM_GIT/playground/terraform/environment/$ENVIRONMENT_NAME/common.tfvars 
+```
+And put the following:
+```
+playground_deploy_sa = "DEPLOY_SA_NAME"
+playground_update_sa = "UPDATE_SA_NAME"
+playground_ci_sa = "CI_SA_NAME"
+playground_cd_sa = "CD_SA_NAME"
+project_id = "PROJECT_ID"
+playground_environment_name = "ENVIRONMENT_NAME"
```

Comment:
Remove all environment specific variables from the list. Set default values instead:
ENVIRONMENT_NAME  =  "environment"
image_tag = "tag"
playground_zone = "fqdn.playground.zone
playground_region = "us-central1"
playground_location = "us-central1-a"
playground_service_account = "pg-gke-sa-env" (SA name length is limited to 25 symbols)
skip_appengine_deploy  = false
state_bucket = "playground-tf-state-environment"

Reply:
What do you mean by environment specific?